cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/controller/ConfirmationSender.kt

```diff
@@ -0,0 +1,41 @@
+package com.cognifide.cogboard.config.controller
+
+import com.cognifide.cogboard.CogboardConstants
+import io.vertx.core.Vertx
+import io.vertx.core.json.JsonObject
+import io.vertx.core.logging.Logger
+import io.vertx.core.logging.LoggerFactory
+
+class ConfirmationSender(private val vertx: Vertx) {
+
+    fun confirmationAfter(action: (JsonObject) -> Boolean, body: JsonObject) =
+            if (action(body)) {
+                sendOk()
+            } else {
+                sendError(body)
+            }
+
+    private fun sendOk() {
+        vertx.eventBus().send(CogboardConstants.EVENT_SEND_MESSAGE_TO_WEBSOCKET,
+                JsonObject().message(OK_MESSAGE))
+    }
+
+    private fun sendError(body: JsonObject) {
+        LOGGER.error("$ERROR_MESSAGE \nconfig:\n$body")
+        vertx.eventBus().send(CogboardConstants.EVENT_SEND_MESSAGE_TO_WEBSOCKET,
+                JsonObject().message(ERROR_MESSAGE))
+    }
+
+    private fun JsonObject.message(message: String): JsonObject {
+        return this
+                .put(CogboardConstants.PROP_EVENT_TYPE, PROP_EVENT_TYPE_NOTIFICATION_CONFIG_SAVE)
+                .put("message", message)
```

Comment:
Consistency: you reference a lot of constants, but here you went with a string literal. I'd put it into a constant as well, it looks weird.